apps/project/schema.py

```diff
@@ -34,13 +65,23 @@ class Meta:
             graphene.Enum.from_enum(PP.Permission),
         ), required=True
     )
+    number_of_leads = graphene.Field(graphene.Int)
+    number_of_leads_tagged = graphene.Field(graphene.Int)
+    number_of_leads_tagged_and_verified = graphene.Field(graphene.Int)
+    number_of_entries = graphene.Field(graphene.Int)
+    number_of_users = graphene.Field(graphene.Int)
+    leads_activity = graphene.List(DateCountType)
+    entries_activity = graphene.List(DateCountType)
+    activity_log = generic.GenericScalar()
+    top_sourcers = graphene.List(EntityDataType)
+    top_taggers = graphene.List(EntityDataType)
```

Comment:
Let's move this to stats = graphene.Field(ProjectStatType)

And define resolve_stats (here use dataloader to calculate annotate stats) and move others resolves there.

Basically, we don't want to calculate all the time, only when users ask for them.